genesis/genesis.go

```diff
@@ -112,7 +112,9 @@ func GenesisStateDiff(
 			return nil, nil, fmt.Errorf("execute function call: %v", err)
 		}
 	}
-	return genesisState.StateDiff(), newClasses, nil
+
+	genesisStateDiff, genesisClasses := genesisState.StateDiff()
```

Comment:
We use `genesisClasses` instead of `newClasses` so we capture the classes declared in function calls, correct?

Reply:
I believe the code is functionally exactly the same, I must have done this just for consistency.